Gems/Atom/RPI/Code/Source/RPI.Public/Pass/Pass.cpp

```diff
@@ -317,6 +317,10 @@ namespace AZ
             // Validate attachmentRef
             if (attachmentRef.m_pass.IsEmpty() || attachmentRef.m_attachment.IsEmpty())
             {
+                AZ_Error("Pass", attachmentRef.m_pass.IsEmpty() && attachmentRef.m_attachment.IsEmpty(),
+                    "Invalid attachment reference [Pass '%s', Attachment '%s']. Both values must be set.",
+                    attachmentRef.m_pass.GetCStr(), attachmentRef.m_attachment.GetCStr());
+
```

Comment:
```suggestion
                AZ_Error("Pass", !attachmentRef.m_pass.IsEmpty() && !attachmentRef.m_attachment.IsEmpty(),
                    "Invalid attachment reference [Pass '%s', Attachment '%s']. Both values must be set.",
                    attachmentRef.m_pass.GetCStr(), attachmentRef.m_attachment.GetCStr());

```

Reply:
Both values will be empty if the file did not include any attachment reference info, which is a valid case, and it's appropriate to just return null. If one is empty and the other is not empty, then the user missed a step and we need to tell them about it.

I can add a code comment about this.